src/main/java/mil/dds/anet/resources/GraphQLResource.java

```diff
@@ -57,26 +74,31 @@
 @PermitAll
 public class GraphQLResource {
 
-	private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+	private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Please user `logger` in lower case as that's also used everywhere else. (keep it consistent).
(Changing it to uppercase because of e.g. Checkstyle settings should be considered a global refactoring.)